README.md

```diff
@@ -1,258 +1,38 @@
-
-## Native Resolutions
-
-If native resolutions are activated you will need a whole new set of modelines:
-
-- 256 x 240 @ 50.006977 SNESpal
-- 256 x 448 @ 50.006977 SNESpal
-- 512 x 224 @ 50.006977 SNESpal
-- 512 x 240 @ 50.006977 SNESpal
-- 512 x 448 @ 50.006977 SNESpal
-- 256 x 240 @ 60.098812 SNESntsc
-- 256 x 448 @ 60.098812 SNESntsc
-- 512 x 240 @ 60.098812 SNESntsc
-- 512 x 224 @ 60.098812 SNESntsc
-- 512 x 448 @ 60.098812 SNESntsc
-- 256 x 192 @ 59.922745 MDntsc
-- 256 x 224 @ 59.922745 MDntsc
-- 320 x 224 @ 59.922745 MDntsc
-- 320 x 240 @ 59.922745 MDntsc
-- 320 x 448 @ 59.922745 MDntsc
-- 320 x 480 @ 59.922745 MDntsc
-- 256 x 192 @ 49.701458 MDpal
-- 256 x 224 @ 49.701458 MDpal
-- 320 x 224 @ 49.701458 MDpal
-- 320 x 240 @ 49.701458 MDpal
-- 320 x 288 @ 49.701458 MDpal
-- 320 x 448 @ 49.701458 MDpal
-- 320 x 480 @ 49.701458 MDpal
-- 320 x 576 @ 49.701458 MDpal
-- 256 x 288 @ 49.701458 MSYSpal
-- 256 x 240 @ 60.098812 NESntsc
-- 256 x 240 @ 50.006977 NESpal
-
-- 640 x 237 @ 60.130001 N64ntsc
-- 640 x 240 @ 60.130001 N64ntsc
-- 640 x 480 @ 60.130001 N64ntsc
-- 640 x 288 @ 50.000000 N64pal
-- 640 x 480 @ 50.000000 N64pal
-- 640 x 576 @ 50.000000 N64pal
-
-- 256 x 252 @ 49.759998 PSXpal
-- 320 x 252 @ 49.759998 PSXpal
-- 384 x 252 @ 49.759998 PSXpal
-- 640 x 252 @ 49.759998 PSXpal
-- 640 x 540 @ 49.759998 PSXpal
-
-- 384 x 240 @ 59.941002 PSXntsc
-- 256 x 480 @ 59.941002 PSXntsc
-
-- 352 x 240 @ 59.820000 Saturn/SGFX_NTSCp
-- 704 x 240 @ 59.820000 SaturnNTSCp
-- 352 x 480 @ 59.820000 SaturnNTSCi
-- 704 x 480 @ 59.820000 SaturnNTSCi
-- 352 x 288 @ 49.701458 SaturnPALp
-- 704 x 288 @ 49.701458 SaturnPALp
-- 352 x 576 @ 49.701458 SaturnPALi
-- 704 x 576 @ 49.701458 SaturnPALi
-
-- 240 x 160 @ 59.730000 GBA
-- 320 x 200 @ 60.000000 Doom
-
-// Arcade
-
-- 400 x 254 @ 54.706841 MK
-- 384 x 224 @ 59.637405 CPS1
-
-These modelines are more accurate giving exact hz. However, some games may have unwanted results. This is due to mid-scanline resolution changes on the original hardware. For the best results super resolutions are the way to go.
-
-## CRT resolution switching & MAME
-
-Some arcade resolutions can be very different from consumer CRTs. There is resolution detection to ensure MAME games will be displayed in the closest available resolution but drawn at their native resolution within this resolution. Meaning that the MAME game will look just like the original hardware.
-
-MAME ROMs that run in a vertical aspect like DoDonPachi need to be rotated within MAME before resolution switching and aspect correction will work. Do this before enabling CRT SwitchRes so that RetroArch will run in your desktop resolution. Once you have rotated any games that may need it turn CRT SwitchRes on.
+### RetroArchWiiRetrode

```

Comment:
This is mistakenly replacing the original README for RetroArch with one about the Retrode driver.

Reply:
Fixed.